src/docker-utils.coffee

```diff
@@ -204,8 +204,7 @@ do ->
 						docker.getImage(tag).removeAsync()
 						.then ->
 							console.log('Deleted image:', tag, image.Id, image.RepoTags)
-						.catch (err) ->
-							console.log('Error deleting image:', tag, image.Id, image.RepoTags, err)
+						.catch (err) -> # Ignore image delete error
```

Comment:
I just feel it makes it easier to read ([example](https://github.com/resin-io/resin-wifi-connect/blob/master/src/app.coffee#L149)), but it's up to you.